packages/a11y/src/test/index.test.js

```diff
@@ -0,0 +1,102 @@
+import { setup, speak } from '../';
+
+jest.mock( '../clear', () => {
+	return jest.fn();
+});
+jest.mock( '@wordpress/dom-ready', () => {
+	return jest.fn( ( callback ) => { callback(); });
+});
+jest.mock( '../filterMessage', () => {
+	return jest.fn( ( message ) => { return message; });
+});
+
+import clear from '../clear';
+import domReady from '@wordpress/dom-ready';
+import filterMessage from '../filterMessage';
+
+describe( 'speak', () => {
+	let containerPolite = document.getElementById( 'a11y-speak-polite' );
+	let containerAssertive = document.getElementById( 'a11y-speak-assertive' );
+
+	beforeEach( () => {
```

Comment:
Should the function spacing exception apply here?

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls